addr2line.go

```diff
@@ -39,25 +39,25 @@ const (
 )
 
 type Workload struct {
-	Addr2ln_offset uint64
+	Query_args     interface{}
 	Addr2ln_name   string
 	Query_str      string
-	Query_args     interface{}
+	Addr2ln_offset uint64
 	Workload_type  Workload_Type
 }
 
```

Comment:
Adjust structs considering the padding that is placed for alignment reason could be a good idea.
As far as I can tell, an interface is laid out as a pointer. In 64-bit architectures, this corresponds to a `uint64`.
This change should not have any impact on the struct size.
https://go.dev/play/p/iXGfeEAwVGU

Reply:
That is very much possible. I am not doing the ordering manually, I'm using `fieldalignment` tool to automatically align fields of all structs in a given package, so these changes are a result of the tool's internal rules of ordering.